articles/data-factory/control-flow-expression-language-functions.md

```diff
@@ -210,7 +210,7 @@ This [tutorial](https://azure.microsoft.com/mediahandler/files/resourcefiles/azu
 |json|Convert the parameter to a JSON type value. It is the opposite of string(). For example, the following expression returns `[1,2,3]` as an array, rather than a string:<br /><br /> `json('[1,2,3]')`<br /><br /> Likewise, you can convert a string to an object. For example, `json('{"bar" : "baz"}')` returns:<br /><br /> `{ "bar" : "baz" }`<br /><br /> **Parameter number**: 1<br /><br /> **Name**: String<br /><br /> **Description**: Required. The string that is converted to a native type value.<br /><br /> The json function supports xml input as well. For example, the parameter value of:<br /><br /> `<?xml version="1.0"?> <root>   <person id='1'>     <name>Alan</name>     <occupation>Engineer</occupation>   </person> </root>`<br /><br /> is converted to the following json:<br /><br /> `{ "?xml": { "@version": "1.0" },   "root": {     "person": [     {       "@id": "1",       "name": "Alan",       "occupation": "Engineer"     }   ]   } }`|  
 |float|Convert the parameter argument to a floating-point number. For example, the following expression returns `10.333`:  `float('10.333')`<br /><br /> **Parameter number**: 1<br /><br /> **Name**: Value<br /><br /> **Description**: Required. The value that is converted to a floating-point number.|  
 |bool|Convert the parameter to a Boolean. For example, the following expression returns `false`:  `bool(0)`<br /><br /> **Parameter number**: 1<br /><br /> **Name**: Value<br /><br /> **Description**: Required. The value that is converted to a boolean.|  
-|coalesce|Returns the first non-null object in the arguments passed in. Note: an empty string is not null. For example, if parameters 1 and 2 are not defined, this returns `fallback`:  `coalesce(pipeline().parameters.parameter1', pipeline().parameters.parameter2 ,'fallback')`<br /><br /> **Parameter number**: 1 ... *n*<br /><br /> **Name**: Object*n*<br /><br /> **Description**: Required. The objects to check for `null`.|  
+|coalesce|Returns the first non-null object in the arguments passed in. Note: an empty string is not null. For example, if parameters 1 and 2 are not defined, this returns `fallback`:  `coalesce(pipeline().parameters?.parameter1', pipeline().parameters?.parameter2 ,'fallback')`<br /><br /> **Parameter number**: 1 ... *n*<br /><br /> **Name**: Object*n*<br /><br /> **Description**: Required. The objects to check for `null`.|  
```

Comment:
We should explain what the '?' does. This may need to be a different section, but for now can you explain that the '?' means null if it doesn't exist

Reply:
If that's directed at me, sorry but I do not know, The initial issue was raised because our customer had an issue and reported that this change resolved it. Please have the appropriate SMEs review. (I'm in Azure support, you can contact me at v-dibr if needed but I don't have much more info.)

Reply:
Hey Dillon, is it alright if we close this PR? I don't think this is the appropriate change. I will update the docs to note the '?' notation

Thanks,
Daniel

Reply:
@djpmsft sure I'm just passing on feedback so if there is a better solution that's fine. Thanks for investigating!